variables.tf

```diff
@@ -83,6 +83,12 @@ variable "container_port" {
   type        = "string"
 }
 
+variable "nlb_healthcheck_port" {
+  description = "The port on which the container will receive a health check from an NLB."
```

Comment:
Can you update the description describing that the setting this to 0 means using the `container_port` as the health check port.

Reply:
sure thing